### PR TITLE
Intellij

### PR DESCRIPTION
**Pull request by nickrodgers42**

### CODE REVIEW

**Comment by levine1726** [comment]
Now this is epic.

File: intellij/.ideavimrc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+"" Source your .vimrc
+"source ~/.vimrc
+scriptencoding utf-8
+set encoding=utf-8
+syntax on
+
+set exrc
+set noerrorbells
```
Comment (levine1726) [inline intellij/.ideavimrc:8]: Absolutely game changing.

File: intellij/.ideavimrc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+set backspace=indent,eol,start
+set ruler
+set laststatus=2
+set confirm
+set visualbell
+set t_vb=
+
+set ttimeoutlen=100
+set updatetime=50
+imap jj <Esc>
```
Comment (levine1726) [inline intellij/.ideavimrc:41]: Nit: I personally find `jk` to be a more satisfying way of escaping insert mode, personally.
```
imap jk <Esc>
```

**Review by levine1726 — COMMENTED** [review]
LGTM overall. Nice work!

File: intellij/.ideavimrc (hunk 2 above)
Comment (nickrodgers42) [reply intellij/.ideavimrc:41]: Addressed in 67113d2d89f386a8162eebe487f986c8d55c669a

**Pull request merged**